dev_settings.py

```diff
@@ -91,6 +91,13 @@
 redis_cache = {
     'BACKEND': 'django_redis.cache.RedisCache',
     'LOCATION': 'redis://127.0.0.1:6379/0',
+    # match production settings
+    'PARSER_CLASS': 'redis.connection.HiredisParser',
+    'REDIS_CLIENT_KWARGS': {
+        'health_check_interval': 15,
+    },
+    # see `settingshelper.update_redis_location_for_tests`
+    'TEST_LOCATION': 'redis://127.0.0.1:6379/1',
```

Comment:
:+1: good idea. I wasn't sure if extra values were allowed in the config, but this is great if they are.